components/calendar/demo/locale.md

```diff
@@ -1,10 +1,18 @@
 ---
 order: 4
-title: 国际化
+title: 
+  zh-CN: 国际化
+  en-US: locale
 ---
 
+## zh-CN
+
 通过 `locale` 配置时区、语言等, 默认支持 en_US, zh_CN
 
+## en_US
+
+To set the properties like time zone, language and so on.And en_US, zh_CN are suppoted in default.
```

Comment:
To set the properties like time zone, language and so on. en_US, zh_CN are suppoted by default.